src/main/java/com/communityratesgames/Application.java

```diff
@@ -9,6 +9,6 @@
     public static void main(String[] args) {
 
         SpringApplication.run(Application.class, args);
-        System.out.println("Hallå eller gubbar!");
+        System.out.println("###Code Compiled Sucessfully###");
```

Comment:
Is this neccesary? Spring boot already says OK if it has been deployed properly.

Reply:
feel free to remove diz